api/controllers/recipes.controller.js

```diff
@@ -60,20 +61,28 @@ exports.getSingleRecipeFromUrl = async(req, res) => {
  * endpoint: /
  */
 exports.createRecipe = async(req, res, next) => {
+
+    let url = await RecipeFormService.buildUrl(req.body.title);
+    let ingredients = await RecipeFormService.formatIngredients(req.body.ingredients);
+    let directions = await RecipeFormService.formatDirections(req.body.directions);
+    let keywords = await RecipeFormService.formatKeywords(req.body.keywords);
+
     const recipe = {
-        name: req.body.name,
-        url: req.body.url,
+        name: req.body.title,
+        url: url,
```

Comment:
You can now use just `url`, instead of `url: url` if the variable is the same name as the key.